src/material/core/tokens/m2/mat/_list.scss

```diff
@@ -12,7 +12,10 @@ $prefix: (mat, list);
 
 // Tokens that can be configured through Angular Material's color theming API.
 @function get-color-tokens($theme) {
-  @return ();
+  @return (
+    active-indicator-color: transparent,
+    active-indicator-shape: 0,
```

Comment:
The shape is usually in the base tokens.

Reply:
Yeah, both of these should be in the base actually, since they don't depend on the theme at all and are constant

Reply:
done, good catch